From: =?utf-8?b?IkjDpXZhcmQgRi4gQWFzZW4i?= <havard.f.aasen@pfft.no>
Date: Fri, 13 Jan 2023 11:23:53 +0100
Subject: use local mathjax

Forwarded: not-needed
---
 docs/conf.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/docs/conf.py b/docs/conf.py
index 318d56b..8246d4c 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -56,6 +56,8 @@ if os.environ.get('READTHEDOCS', 0):
     sys.modules['wand.version'].QUANTUM_SCALE = None
     sys.modules['wand.version'].MAGICK_HDRI = None
 
+mathjax_path = 'file:///usr/share/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML'
+
 # -- General configuration ----------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
